README.md

```diff
@@ -406,6 +405,12 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 
 #### Transformers
 
+- [vite-plugin-swc-only](https://github.com/natrim/vite-plugin-swc-only) - Use SWC to transform and minify app.
+
+#### Helpers
+
+- [vite-plugin-react-refresh](https://github.com/fengxinming/vite-plugins/tree/main/packages/vite-plugin-react-refresh) - Provide enhancements for [@vitejs/plugin-react-refresh](https://www.npmjs.com/package/@vitejs/plugin-react-refresh).
```

Comment:
I don’t think it makes sense to single out a separate category, perhaps a subcategory is better

Reply:
?? it's not new category, its the same subcategory as Vue has above

Reply:
I'm talking about analogue https://github.com/vitejs/awesome-vite#react-1